docs/assets/js/tabs.js

```diff
@@ -1,52 +1,259 @@
-import { Tabs } from 'govuk-frontend';
-
-/**
- * @param {DOMNode} container - Parent element containing Gov UK tab HTML
- */
-export function init( container ) {
-  const tabs = container.querySelectorAll( '[data-module="tabs"]' );
-
-  if ( tabs && tabs.length > 0 ) {
-    for ( let i = 0; i < tabs.length; i++ ) {
-      const tab = tabs[i];
-      new Tabs( tab ).init();
+const BASE_CLASS = 'm-tabs';
+
+function Tabs() {
+  const tablist = document.querySelectorAll('[role="m-tabs__list"]')[0];
+  let tabs;
+  let panels;
+  // const delay = determineDelay();
+
+  generateArrays();
+
+  function generateArrays () {
+    tabs = document.querySelectorAll('[role="m-tabs__tab"]');
+    panels = document.querySelectorAll('[role="m-tabs__panel"]');
+  };
+
+  // For easy reference
+  const keys = {
+    end: 35,
+    home: 36,
+    left: 37,
+    up: 38,
+    right: 39,
+    down: 40,
+    delete: 46
+  };
+
+  // Add or substract depending on key pressed
+  const direction = {
+    37: -1,
+    38: -1,
+    39: 1,
+    40: 1
+  };
+
+  // Bind listeners
+  for (let i = 0; i < tabs.length; ++i) {
```

Comment:
Should go inside init or it's own `_bindListeners()` private method. Remember the `_` prefix just means a method that is not exposed externally by the class at the end of the file with this.method = method.